PennMobile/Laundry/Model/LaundryNotificationCenter.swift

```diff
@@ -8,8 +8,10 @@
 
 import Foundation
 import UserNotifications
-import ActivityKit
 import PennMobileShared
+#if canImport(ActivityKit)
+import ActivityKit
+#endif
```

Comment:
This will technically still crash if someone tries to start a live activity on mac, but it fixes the crash on startup

Reply:
I'm assuming that it was that commit, but i don't actually know